validation.js

```diff
@@ -0,0 +1,12 @@
+const submitBtn = document.querySelector('.contact-button');
+const validateEmail = (email) => email === email.toLowerCase();
+
+submitBtn.addEventListener('click', () => {
+  const emailVal = document.querySelector('.email').value;
+
+  if (!validateEmail(emailVal)) {
+    window.alert(
+      'Please enter email as lower case eg. shuhad.loofer@gmail.com',
+    );
+  }
+});
```

Comment:
- [ ]  You should implement your error message in your browser, not as an alert message. Check in the project requirement and implement what is suggested. 
![image](https://user-images.githubusercontent.com/43877689/175486419-2cf9f5ca-a7ff-4d80-ad5e-ce9acf884e67.png)
![image](https://user-images.githubusercontent.com/43877689/175486810-fd33d0ea-bb3b-47a6-8e0a-457beed8d06f.png)